src/main/java/net/sf/jabref/cli/ArgumentProcessor.java

```diff
@@ -208,7 +211,7 @@ private boolean exportMatches(List<ParserResult> loaded) {
                             matches);
                 } catch (Exception ex) {
                     System.err.println(Localization.lang("Could not export file") + " '" + data[1] + "': "
-                            + ex.getMessage());
+                            + ExceptionUtils.getStackTrace(ex));
```

Comment:
Yes, because if you look into the issue, ex.getMessage simply returned "null" and there was no stacktrace visible, which made it hard to locate the bug.
`Could not export file 'references.html': null`